src/usr/local/www/system_advanced_admin.php

```diff
@@ -597,4 +597,4 @@
 	log_error(gettext("webConfigurator configuration has changed. Restarting webConfigurator."));
 	send_event("service restart webgui");
 }
-?>
\ No newline at end of file
+?>
```

Comment:
Seems like your editor has added a new line after the closing tag. This is a bad idea as per:
https://secure.php.net/manual/en/language.basic-syntax.phptags.php
> If a file is pure PHP code, it is preferable to omit the PHP closing tag at the end of the file. This prevents accidental whitespace or new lines being added after the PHP closing tag, which may cause unwanted effects because PHP will start output buffering when there is no intention from the programmer to send any output at that point in the script.```